src/lib.rs

```diff
@@ -494,12 +494,21 @@ unsafe extern "C" fn dummy_free(data: *const u8, user_data: *mut c_void) {
     }
 }
 
+unsafe fn dav1d_parse_sequence_header_error(
```

Comment:
```suggestion
unsafe fn rav1d_parse_sequence_header_error(
```

I've been naming `fn`s that use `Rav1d*` types `rav1d_*` and `fn`s that use `Dav1d*` types `dav1d_*`.